server/utils/utils.ts

```diff
@@ -88,6 +91,19 @@ export const formatAssessmentData = (
           score = area.upperBound
         }
       }
+
+      if (crimNeeds.lifestyleAndAssociates && area.crimNeedsKey === 'thinkingBehaviourAndAttitudes') {
+        subData = {
+          upperBound: '6',
+          thresholdValue: '1',
+          criminogenicNeedsScore: crimNeeds.lifestyleAndAssociates.lifestyleOtherWeightedScore,
+        }
+        overallScore = Math.max(
+          Number(crimNeeds.thinkingBehaviourAndAttitudes.thinkOtherWeightedScore),
+          Number(subData.criminogenicNeedsScore),
+        )
+      }
```

Comment:
Null checks needed here? Seems like you might be able to cause an exception if `thinkingBehaviourAndAttitudes` was null for example, and then you tried turn null into a number (though maybe it can handle this)

Reply:
Added check for `thinkingBehaviourAndAttitude`. 

`Math.max(null, null)` = `0`